addons/mail/static/src/discuss_app/im_status.xml

```diff
@@ -2,12 +2,13 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ImStatus" owl="1">
-        <div class="o-mail-ImStatus position-absolute top-100 start-100 translate-middle w-50 h-50 d-flex justify-content-center flex-shrink-0 align-items-center mt-n1 ms-n1 rounded-circle small o-bg-inherit" t-att-class="props.className">
-            <span t-if="props.persona.im_status" class="d-flex flex-column">
+        <div class="o-mail-ImStatus d-flex justify-content-center flex-shrink-0 align-items-center rounded-circle o-bg-inherit" t-att-class="props.className" t-att-style="props.style">
+            <span t-if="props.persona.im_status" class="d-flex flex-column" name="icon">
                 <i t-if="props.persona.im_status === 'online'" class="fa fa-circle text-success" title="Online" role="img" aria-label="User is online"/>
-                <i t-if="props.persona.im_status === 'away'" class="fa fa-circle text-warning" title="Idle" role="img" aria-label="User is idle"/>
-                <i t-if="props.persona.im_status === 'offline'" class="fa fa-circle-o text-700" title="Offline" role="img" aria-label="User is offline"/>
-                <i t-if="props.persona.im_status === 'bot'" class="fa fa-heart text-success" title="Bot" role="img" aria-label="User is a bot"/>
+                <i t-elif="props.persona.im_status === 'away'" class="fa fa-circle text-warning" title="Idle" role="img" aria-label="User is idle"/>
+                <i t-elif="props.persona.im_status === 'offline'" class="fa fa-circle-o text-700" title="Offline" role="img" aria-label="User is offline"/>
+                <i t-elif="props.persona.im_status === 'bot'" class="fa fa-heart text-success" title="Bot" role="img" aria-label="User is a bot"/>
+                <i t-else="" class="fa fa-fw fa-question-circle" title="No IM status available"/>
```

Comment:
I think this change (using `else` rather than named `if`s) is a regression (see comment in HR override).

Reply:
I don't see how this is a regression. Using `elif` ensures we don't introduce a bug to show multiple icons, and the fallback manages all cases were we cannot show a specific icon for im_status. This implicitly handles strange im_status like `im_partner`